<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QListView" name="listView"/>
       <widget class="QTableView" name="tableView"/>
       <widget class="QTextEdit" name="LogList">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>85</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTextEdit" name="commandEdit"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="clearButton">
              <property name="minimumSize">
               <size>
                <width>91</width>
                <height>61</height>
               </size>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="submitButton">
              <property name="minimumSize">
               <size>
                <width>91</width>
                <height>61</height>
               </size>
              </property>
              <property name="text">
               <string>Submit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="ErrorStr">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>59</height>
           </size>
          </property>
          <property name="text">
           <string>                                                           Errors</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actioninfo"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actioninfo">
   <property name="text">
    <string>info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
